contracts/VaultV3.vy

```diff
@@ -662,52 +638,16 @@ def _max_withdraw(
     return max_assets
 
 @internal
-def _deposit(sender: address, recipient: address, assets: uint256) -> uint256:
+def _deposit(sender: address, recipient: address, assets: uint256, shares: uint256):
     """
-    Used for `deposit` calls to transfer the amount of `asset` to the vault, 
-    issue the corresponding shares to the `recipient` and update all needed 
+    Used for `deposit` and `mint` calls to transfer the amount of `asset` to the vault, 
+    issue the corresponding `shares` to the `recipient` and update all needed 
     vault accounting.
     """
-    assert self.shutdown == False # dev: shutdown
```

Comment:
why remove this?

Reply:
It is unnecesary.

When the vault is shutdown the deposit limit module is always set to address(0) and deposit limit to 0. https://github.com/yearn/yearn-vaults-v3/blob/411cae31a2e7cdae49364bdbfa3f601667ef141e/contracts/VaultV3.vy#L1772

And cannot be changed anymore. 

So _max_deposit() will always return 0, which will revert on the assertions